static/js/components/widgets/RelationField.test.tsx

```diff
@@ -249,6 +249,27 @@ describe("RelationField", () => {
           label: item.title
         }))
       )
+
+      await act(async () => {
+        // @ts-ignore
+        wrapper.find("SortableSelect").prop("onChange")(
+          // @ts-ignore
+          [contentListingItems[0].text_id]
+        )
```

Comment:
**Requested Change:** If you pass the `SortableSelect` function to `find` instead of the string then ts will know that (a) the prop is required and (b) the argument is `string[]` not ChangeEvent. Then we can get rid of both the `@ts-ignore`.

Reply:
ah nice catch! just made that change